descriptors/ModuleDescriptor-template.json

```diff
@@ -47,6 +47,32 @@
           "pathPattern": "/_/tenant"
         }
       ]
+    },
+    {
+      "id": "central-servers",
```

Comment:
Although you can potentially separate different groups of endpoints, I'd suggest to combine them under singer interface unless there is strong reason to not do it. 

So it can be potentially one "inn-reach" interface to hold everything, or it might be "inn-reach-configuration" /  "inn-reach-settings" to place everything related to configuration under it